### ISSUE TITLE
The Fab apply icon missing

### ISSUE THREAD

**Issue by SamNeill** [jahirfiquitiva/Blueprint#249]
- [x] I have verified there are no duplicate active or recent bugs, questions, or requests
- [x] I have verified that I am using the latest version.

### Device/App info:
 - Blueprint Version: `2.1.4`
 - Android version: `Android 11`
 - Device Manufacturer: `Google`
 - Device Name: `Pixel 3 XL`

### Describe the bug
fab apply icon missing  and nova launcher isn't installed or any others launcher. 

### Reproduction Steps
1. Download the latest version of blueprint 2.1.4
2. Open in android studio and started from scratch. 
3. Let it fully sync with Net on

### Expected behavior

The apply icon should be there for apply and installed launcher should show up in list.

### Screenshots

<p>
<img src="https://user-images.githubusercontent.com/23134828/98704511-8f3ee200-239e-11eb-8442-6580aa5ade43.png" width="300"/>
<img src="https://user-images.githubusercontent.com/23134828/98704518-91a13c00-239e-11eb-83bb-1439d6b7f102.png" width="300"/>
</p>

### Code and/or Logs

```gradle
    java.lang.SecurityException: Permission denial: writing to settings requires:android.permission.WRITE_SECURE_SETTINGS
        at com.android.providers.settings.SettingsProvider.enforceWritePermission(SettingsProvider.java:2065)
        at com.android.providers.settings.SettingsProvider.mutateGlobalSetting(SettingsProvider.java:1283)
        at com.android.providers.settings.SettingsProvider.insertGlobalSetting(SettingsProvider.java:1237)
        at com.android.providers.settings.SettingsProvider.call(SettingsProvider.java:395)
        at android.content.ContentProvider.call(ContentProvider.java:2448)
        at android.content.ContentProvider$Transport.call(ContentProvider.java:517)
        at android.content.ContentProviderNative.onTransact(ContentProviderNative.java:295)
        at android.os.Binder.execTransactInternal(Binder.java:1154)
        at android.os.Binder.execTransact(Binder.java:1123)
```

**Comment by jahirfiquitiva** [comment]
Please add a screenshot where it's shown Nova launcher is installed and its package name

**Comment by SamNeill** [comment]
Screenshot is already posted the second one and the package name is
com.sam.nepal.colors
All letters are small.

**Comment by SamNeill** [comment]
![Screenshot_20201110-223125](https://user-images.githubusercontent.com/23134828/98709683-87823c00-23a4-11eb-9e40-5b1fbb5f4910.png)

**Comment by jahirfiquitiva** [comment]
@SamNeill can you add a screenshot of the installed apps, in settings, where nova is shown as installed, and also its package name (Nova's, not your app's)

**Comment by SamNeill** [comment]
![Screenshot_20201111-065413](https://user-images.githubusercontent.com/23134828/98755665-dce13c00-23ea-11eb-87db-1fb59252e660.png)
![Screenshot_20201111-065434](https://user-images.githubusercontent.com/23134828/98755672-df439600-23ea-11eb-8e2d-c6d386f996ea.png)

**Comment by SamNeill** [comment]
![Screenshot_20201111-150539](https://user-images.githubusercontent.com/23134828/98798360-a11c9580-242f-11eb-9518-6a26b8ac6ca4.png)
![Screenshot_20201111-150546](https://user-images.githubusercontent.com/23134828/98798396-aaa5fd80-242f-11eb-9dca-5b1a56ac6818.png)
Hi Jahir here is the screenshot with latest blueprint but with target SDK version 29..

**Comment by RandleP** [comment]
Can confirm no launcher in the apply list shows up as installed with 2.1.4
I have Lawnchair, Nova, Smart and Sqaure Home installed.

**Comment by jahirfiquitiva** [comment]
Fixed in v `2.1.5`
